examples/multiProcessExample/src/main/java/io/realm/examples/realmmultiprocessexample/MainActivity.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright 2014 Realm Inc.
```

Comment:
2017 in this file and a bunch of others

Reply:
UH, yeah! This code was written in 2014 actually ! will change it.